stats.py

```diff
@@ -419,6 +436,46 @@ def get_comments(revision: DiffDb.Revision, sessions: Sessions) -> list[dict]:
     return comments
 
 
+def get_transactions(revision: DiffDb.Revision, sessions: Sessions) -> list[dict]:
+    # If the newValue or oldValue is a boolean, then we convert it to the
+    # string "1" or "0". Otherwise we leave it as a string.
+    def format_value(value):
+        if isinstance(value, bool):
+            return str(int(value))  # "1" for True, "0" for False
+        if isinstance(value, str):
+            return value  # leave strings unchanged
+        return str(value)  # fallback: convert everything else to string
+
+    transactions = []
+
+    for transaction in (
+        sessions.diff.query(DiffDb.Transaction)
+        .filter(
+            DiffDb.Transaction.objectPHID == revision.phid,
+            DiffDb.Transaction.transactionType.in_(STATE_CHANGE_TYPES),
+        )
+        .order_by(DiffDb.Transaction.dateCreated)
+    ):
+
+        newValue = format_value(transaction.newValue)
+        oldValue = format_value(transaction.oldValue)
```

Comment:
Always use snake-case for variable names in Python:

```suggestion
        new_value = format_value(transaction.newValue)
        old_value = format_value(transaction.oldValue)
```

The SQLAlchemy objects in the phab-etl use camel-case, however that's due to those attribute names being derived from the database schema, so we can't change them.